chain/client/src/metrics.rs

```diff
@@ -563,6 +563,16 @@ pub(crate) static SHADOW_CHUNK_VALIDATION_FAILED_TOTAL: Lazy<IntCounter> = Lazy:
     .unwrap()
 });
 
+pub(crate) static CHUNK_STATE_WITNESS_VALIDATION_ELAPSED: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_chunk_state_witness_validation_elapsed",
```

Comment:
Is the unit seconds? I think we should be clear here - in the name maybe? Not sure what the convention is.

Reply:
Yes, the unit is seconds. It doesn't look like we follow any conventions, I've found `_time`, `_elapsed`, `_sec`, `_seconds`. Will use `_seconds` here.